go.mod

```diff
@@ -4,7 +4,7 @@ go 1.20
 
 require (
 	github.com/AlekSi/pointer v1.2.0
-	github.com/G-Core/gcore-dns-sdk-go v0.2.5
+	github.com/G-Core/gcore-dns-sdk-go v0.2.6-0.20230519073313-7540d3ccc1f7
```

Comment:
why using version in such format

Reply:
because it wasn't released in gcore-dns-sdk-go master branch